.github/workflows/test.yml

```diff
@@ -35,8 +35,8 @@ jobs:
       - name: Setup testdata
         uses: "./.github/actions/setup_testdata"
 
-      - name: Run fast tests
-        run: pytest --disable-warnings -x --hypothesis-profile ci-fast
+      - name: Run tests
+        run: pytest -n 4 tests --disable-warnings
 
```

Comment:
What is the impact of removing `-x --hypothesis-profile ci-fast`?

Reply:
Makes you use the default hypothesis settings which does some healthchecks which don't work well on github actions runners. In ert we set some permissive default settings instead which work just fine. So you can change

https://github.com/equinor/xtgeo/blob/abe3648037990e9c08f0e813f3abb694c688b437/conftest.py#L7-L15

to 

```python
# Timeout settings are unreliable both on CI and
# when running pytest with xdist so we disable it
settings.register_profile(
    "no_timeouts",
    deadline=None,
    suppress_health_check=[HealthCheck.too_slow],
)
settings.load_profile("no_timeouts")
```

if you want and that should work well both for running xdist locally and on ci. There is no need to use the `--hypothesis-profile`  with the above change as `settings.load_profile("no_timeouts")` makes the `no_timeouts` profile the default one.

Reply:
That's exactly where I copied from in this pr 😄

Additionally, `-x` will cause the pytest to stop on the first failed test. That might be useful if the test suite takes a very long time to run but since we're just a few minutes I thought it's more useful to just run on and show all failures.

Reply:
Yes, for ci runs which do not take too long, dropping `-x` can be useful to see all failures, and not just the first one.